src/Compilers/CSharp/Test/Emit2/Emit/NumericIntPtrTests.cs

```diff
@@ -8129,7 +8129,7 @@ void binaryOperator(string op, string leftType, string rightType, string expecte
 
                 if (expectedDiagnostics.Length == 0)
                 {
-                    CompileAndVerify(comp, verify: Verification.FailsPEVerify);
+                    CompileAndVerify(comp, verify: Verification.FailsPEVerify, emitOptions: EmitOptions.Default);
```

Comment:
>emitOptions: EmitOptions.Default [](http://example.com/codeflow?start=79&length=32)

What effect does this change have?

Reply:
This is a different perf fix I was working on that I accidentally included with this change. I'll revert this part